app/src/main/java/com/keylesspalace/tusky/components/login/LoginWebViewActivity.kt

```diff
@@ -127,19 +127,29 @@ class LoginWebViewActivity : BaseActivity(), Injectable {
             }
         }
         webView.setBackgroundColor(Color.TRANSPARENT)
-        webView.loadUrl(data.url.toString())
+
+        if (savedInstanceState == null) {
+            webView.loadUrl(data.url.toString())
+        } else {
+            webView.restoreState(savedInstanceState)
```

Comment:
never had to do this before but it doesn't hurt I guess

Reply:
Without it state is lost on rotation and user has to enter password again